src/lib.rs

```diff
@@ -328,33 +327,68 @@ impl LocalExecutor {
     /// assert_eq!(res, 6);
     /// ```
     pub fn run<T>(&self, future: impl Future<Output = T>) -> T {
-        pin!(future);
+        self.enter(|| run(future))
+    }
+}
+
+impl Default for LocalExecutor {
+    fn default() -> LocalExecutor {
+        LocalExecutor::new()
+    }
+}
+
+/// Runs both Executor and LocalExecutor
+fn run<T>(future: impl Future<Output = T>) -> T {
+    // If local executor is initialized, then use its parker
+    let p = if LOCAL_EX.is_set() {
+        LOCAL_EX.with(|local_ex| Rc::clone(&local_ex.parker))
+    } else {
+        Rc::new(parking::Parker::new())
+    };
```

Comment:
I couldn't found another way without refactoring `multitask` and introducing `LocalTicker`.